en/extend_your_application/README.md

```diff
@@ -134,7 +134,7 @@ Oh no! Another error! But we already know how to deal with it, right? We need to
 
 We will create a file in `blog/templates/blog` called `post_detail.html`, and open it in the code editor.
 
-It will look like this:
+Let's type the following code:
```

Comment:
:+1:

Reply:
Thank you. I have taken out the other file and edited the description of this PR. Please re-review.